<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAssetProperties</class>
 <widget class="QDialog" name="DlgAssetProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asset Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Asset Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblNumSelected">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedAssetType">
     <widget class="QWidget" name="atlasPage">
      <widget class="QComboBox" name="cmbTextureType">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmbTextureFiltering">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="audioPage"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgAssetProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgAssetProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
